frontend/src/css/main.scss

```diff
@@ -387,3 +387,25 @@
     transform: translate(24px, 0);
   }
 }
+
+// Markdown text transition animation
+
+@keyframes nextStepTransition {
+  from {
+    background-color: lightyellow;
+  }
+  to {
+    background-color: inherit;
+  }
+}
+
+// Body transition
+
+@keyframes bodyTransition {
+  from {
+    opacity: 0.5;
```

Comment:
I'm not against an animation for changing the page, but I just tried this and it feels weird. Looks like a flash of lightning or something.

Reply:
True. Now that I look at it, it does seem weird. I added now a sort of 'fade-in' effect which only affects course text (won't affect terminal this time).

Reply:
Page transitions have now been removed.